app/controllers/concerns/sufia/generic_works_controller_behavior.rb

```diff
@@ -15,6 +15,7 @@ module GenericWorksControllerBehavior
     def new
       # TODO: move this to curation_concerns
       @form = form_class.new(curation_concern, current_ability)
+      curation_concern.apply_depositor_metadata(current_user)
```

Comment:
Set `curation_concern.depositor=` instead of `apply_depositor_metadata` the latter creates a bunch of ACLs which aren't needed here.